src/main/java/com/google/api/generator/engine/format/BUILD.bazel

```diff
@@ -0,0 +1,22 @@
+package(default_visibility = ["//visibility:public"])
```

Comment:
As discussed, please move all the comment-related code under `ast` or `ast/comment`.

Reply:
sure, it might make more sense to put in `ast/comment`, thank you!

Reply:
all files are moved to ast/folder, thx!